angular-chess/src/app/services/chess-board.service.spec.ts

```diff
@@ -147,6 +147,18 @@ describe('ChessBoardService', () => {
 
       expect(service.getPlyCount()).toEqual(42);
     });
+
+    it('should set number of moves to 22', () => {
+      service.importFen("rnbqkbnr/pppppppp/8/8/8/8/PPPPPPPP/RNBQKBNR w KQkq - 0 22");
+
+      expect(service.getMoveCount()).toEqual(22);
+    });
+
+    it('should set number of moves to 42', () => {
+      service.importFen("rnbqkbnr/pppppppp/8/8/8/8/PPPPPPPP/RNBQKBNR w KQkq - 0 42");
+
+      expect(service.getMoveCount()).toEqual(42);
+    });
```

Comment:
What is the difference between the 22 and 42 case?
Is it the same? If yes, then is one case enough or add a better description.

Reply:
I implemented it with the "test first strategy". If I add only one test then it would pass if the method would simply return 22. That's why a second test is needed.

Reply:
👍 
One idea to reduce the code redundance:
https://stackoverflow.com/questions/38659880/jasmine-parameterized-unit-test